components/ipfs/common/pref_names.cc

```diff
@@ -0,0 +1,9 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/ipfs/common/pref_names.h"
+
+const char kIPFSResolveMethod[] = "brave.ipfs.resolve_method";
```

Comment:
would be nice to have some comments about the prefs

Reply:
Added comments, thanks for the suggestion 👍